cli/src/format.rs

```diff
@@ -19,10 +19,14 @@ use vulnreach_types::Vulnerability;
 use crate::commands::status::PhylumStatus;
 use crate::print::{self, table_format};
 use crate::types::{
-    HistoryJob, Issue, ListUserGroupsResponse, Package, PolicyEvaluationResponse,
-    PolicyEvaluationResponseRaw, RiskLevel, UserGroup, UserToken,
+    HistoryJob, Issue, ListUserGroupsResponse, OrgMember, OrgMembersResponse, OrgsResponse,
+    Package, PolicyEvaluationResponse, PolicyEvaluationResponseRaw, RiskLevel, UserGroup,
+    UserToken,
 };
 
+// Maximum length of email column.
+const MAX_EMAIL_WIDTH: usize = 25;
```

Comment:
With the understanding that this is a refactor of an existing constant (pulling it up), this width seems low. AFAICT, bumping this to 35 (or even as high as 46) will still limit the total MAX width of the largest use cases to 80.

Reply:
The appropriate email width has been determined to be 25 in all currently relevant cases. There's no reason to change it when this is the best value for what it is used for.